cmd/derper/derper.go

```diff
@@ -293,9 +297,19 @@ func main() {
 		})
 		if *httpPort > -1 {
 			go func() {
+				certHandler := certManager.HTTPHandler(tsweb.Port80Handler{Main: mux})
```

Comment:
or

```go
    port80mux := http.NewServeMux()
    port80mux.HandleFunc("/generate_204", serve204NoContent)
    port80mux.Handle("/", certManager.HTTPHandler(tsweb.Port80Handler{Main: mux}))
```

(and similar HandleFunc serve204NoContent above)

your call either way